temple/update.py

```diff
@@ -32,20 +34,16 @@ def _cookiecutter_configs_have_changed(template, old_version, new_version):
     Returns:
         bool: True if the cookiecutter.json files have been changed in the old and new versions
     """
-    temple.check.is_git_ssh_path(template)
-    repo_path = temple.utils.get_repo_path(template)
-    github_client = temple.utils.GithubClient()
-    api = '/repos/{}/contents/cookiecutter.json'.format(repo_path)
-
-    old_config_resp = github_client.get(api, params={'ref': old_version})
-    old_config_resp.raise_for_status()
-    new_config_resp = github_client.get(api, params={'ref': new_version})
-    new_config_resp.raise_for_status()
+    with tempfile.TemporaryDirectory() as clone_dir:
+        repo_dir = cc_vcs.clone(template, old_version, clone_dir)
```

Comment:
ok, I see why this breaks `test_update.py`. All responses are [mocked out](https://github.com/aclowes/temple/blob/e41a6dee591a9291c03ce26e151de88c0fa75c2f/temple/tests/conftest.py#L26), whereas the vcs options are not yet...I'll look to see if there are mocks available for cookiecutter.vcs